easybuild/easyconfigs/w/WRF/WRF-3.6.1-foss-2015a-dmpar.eb

```diff
@@ -0,0 +1,40 @@
+name = 'WRF'
+version = '3.6.1'
+
+homepage = 'http://www.wrf-model.org'
+description = """The Weather Research and Forecasting (WRF) Model is a next-generation mesoscale
+ numerical weather prediction system designed to serve both operational forecasting and atmospheric
+ research needs."""
+
+toolchain = {'name': 'foss', 'version': '2015a'}
+toolchainopts = {'opt': False}  # don't use agressive optimization, stick to -O2
```

Comment:
Isn't `opt` set to `False` by default? And does this automatically fall back to `-O2`?